### DISCUSSION TITLE
Hidden Field Gives error

### DISCUSSION THREAD

**Discussion by Elshaden** [laravel/nova-issues#4815] [Q&A] [answered]
- Laravel Version: 9.3.1
- Nova Version: 4.14.3
- PHP Version: 8.1.#
- Database Driver & Version:
- Operating System and Version: #.#
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

### Description:
Using the Hidden Field gives error and not working as described

In My Resource :
```
   Hidden::make('User', 'user_id')->default(function ($request) {
                return $request->user()->id;
            }),
```
In Browser Inspect
```
<div class="hidden" errors="[object Object]" index="0"><input type="hidden" value="5"></div>
```

The Field Value is not passed on

**Comment by crynobone** [comment, marked as answer]
The above seems right to me. Can you elaborate on what's exactly wrong.

**Reply by Elshaden** [reply]
The Hidden Field basically does not work.
as you can see above, every thing is correct, yet the user_id does not go to the database table to be created and I get an error that form Sql saying the user_id field is required and cannot be null. 

if I change the same field to Text field it works. but Hidden does not.

**Reply by Elshaden** [reply]
I think that the Hidden field is working fine
somewhere in my code preventing the user_id from reaching the DB
Sorry
